src/test/java/com/imagination/cbs/service/helper/BookingApproveHelperTest.java

```diff
@@ -44,12 +44,14 @@
 
 	@Mock
 	private EmailHelper emailHelper;
+
+	private Team approverTeam;
 	
 	@Test
 	public void shouldPrepareEmailAndSendTO_HR_WhenBookingCanBeOverride() {
 		final Booking booking = createBooking();
-		final ApprovalStatusDm approvalStatus = createApprovalStatusDm();
-		final Team approverTeam = createTeam();
+		final ApprovalStatusDm approvalStatus = createApprovalStatusDm(1004L);
```

Comment:
this method has multiple verify .... can be broken to multiple test cases.

Reply:
Done changes according to your suggestions